poc_iot_verifier/src/poc.rs

```diff
@@ -215,11 +225,14 @@ impl Poc {
                     let gw_info: GatewayInfo = witness_result.gateway_info.ok_or_else(|| {
                         Error::not_found("invalid FollowerGatewayResp for witness")
                     })?;
-                    // TODO: perform hex density check here for a valid witness
+                    let scaling_factor = density_queries
+                        .query(gw_info.location.unwrap_or_default().to_string())
+                        .await?
+                        .unwrap_or(dec!(1.0));
```

Comment:
```suggestion
                        .unwrap_or(Decimal::ONE);
```

Reply:
Nice, i missed this constant

Reply:
[3a99e29](https://github.com/helium/oracles/pull/191/commits/3a99e2995a48b124f65812a2e7cb9b90a425b969)